sdk/react/src/views/AdPlaybackScreen/AdPlaybackScreen.js

```diff
@@ -265,8 +276,8 @@ export default class AdPlaybackScreen extends Component {
         <View style={styles.adContainer}>
           {adBar}
           {this._renderPlaceholder(adIcons)}
-          {this._renderPlayPause(shouldShowControls)}
-          {this._renderBottomOverlay(shouldShowControls)}
+          {this._renderPlayPause(this.state.shouldShowControls)}
```

Comment:
`_renderPlayPause` and `_renderBottomOverlay` have access to the state, no need to pass it as an argument

Reply:
done